src/core/frame/to_arrow.cc

```diff
@@ -345,6 +346,7 @@ Column dt::ColumnImpl::as_arrow() const {
     case SType::INT64: return _as_arrow_fw<int64_t>();
     case SType::FLOAT32: return _as_arrow_fw<float>();
     case SType::FLOAT64: return _as_arrow_fw<double>();
+    case SType::DATE32: return  _as_arrow_fw<int32_t>();
```

Comment:
So we just convert timestamps, right?

Reply:
Yes, our DATE32 format coincides with arrow's `date32`, so this conversion is straightforward.